judge/utils/mail.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template import loader
+
+
+bad_mail_regex = list(map(re.compile, settings.BAD_MAIL_PROVIDER_REGEX))
+
+
+def is_email_address_bad(email):
+    if '@' in email:
+        domain = email.split('@')[-1].lower()
```

Comment:
What does the email RFC say about multiple @ in email addresses?

Reply:
It looks like you can only have multiple @ in the username part of the email address, while the domain part cannot contain a @. I think this should be fine since we can assume anything after the last part is the domain.